Sources/BraintreeCore/BTAppContextSwitcher.swift

```diff
@@ -14,6 +14,11 @@ import UIKit
     /// The URL scheme to return to this app after switching to another app or opening a SFSafariViewController.
     /// This URL scheme must be registered as a URL Type in the app's info.plist, and it must start with the app's bundle ID.
     /// - Note: This property should only be used for the Venmo flow.
+    @available(
+        *,
+        deprecated,
+        message: "returnURLScheme is deprecated and will be removed in a future version. Use BTVenmoClient(apiClient:universalLink:)."
```

Comment:
Should this deprecation go in the CHANGELOG?

Reply:
So I forgot about this swift limitation until now, but we cannot add this message since it causes cocoapods to fail ([previously discussed here](https://github.com/braintree/braintree_ios/pull/1130)). We cannot use the same solution in that PR because `BTAppContextSwitcher` lives in `Core` so the property would need to be public and the solution is to keep it internal. There may be some wonky workarounds but still no out of the box solutions. Any ideas?

Reply:
We could do the same thing as that PR ^, but no-doc the "internal" (actually public) interior property that it references?

Reply:
Ah, you're right, I had also added a deprecation message to the public property which resulted in the same error. 🤦

Reply:
Updated: https://github.com/braintree/braintree_ios/pull/1440/commits/2705af71d15b7b0ee218ad97d9d27db7ff362c70